### ISSUE TITLE
Accessibility: My Account nav unlabeled

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43625]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The navigation menu on the My Account page is unlabeled. See:
https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce/templates/myaccount/navigation.php#L25

### Expected behavior

There should be an aria-label on the nav tag. Something like:
`<nav class="woocommerce-MyAccount-navigation" aria-label="Account Pages">`

### Actual behavior

No label is present.

### Steps to reproduce

See code linked above.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.
